### ISSUE TITLE
Match end sound effect

### ISSUE THREAD

**Issue by Desfile** [Warzone/TGM#658]
**Describe the bug**
At the end of each match, there's an awful loud sound which plays.

**Client version**
1.15.x

**To Reproduce**
Steps to reproduce the bug:
1. Play either in a match or as a spectator.
2. Wait for a match to end.

**Expected behavior**
A softer sound effect.

**Server Software (Spigot/PaperSpigot):**
PaperSpigot

**Comment by nichxlxs** [comment]
Is the problem that the sound is too loud or the sound should be changed to something else?

**Comment by Desfile** [comment]
The sound is too loud. Therefore, I'm recommending either lower the volume of the sound, or change it overall.

**Comment by benrobson** [comment]
> The sound is too loud. Therefore, I'm recommending either lower the volume of the sound, or change it overall.

Is this still a recurring issue since June 2020?

**Comment by Owehttamy** [comment]
Still a recurring issue

**Comment by benrobson** [comment]
In my recent change from just above this, you can change the volume from the Ambient sound category in game. 
Give this a try and let me know how it goes.

**Comment by benrobson** [comment]
Make sure you are operating on the latest commit.

**Comment by Owehttamy** [comment]
Yes, this is still apparent, as seen [here](https://youtu.be/lrK6qlFkoc4) (volume warning, of course)
Just to clarify: the changes did seem to fix the KOTH, CTW, and DTM sounds effects, but they didn't fix the match end sound effect.

**Comment by benrobson** [comment]
Where the match end sound effect is: https://github.com/benrobson/TGM/blob/fix-end-match-sound/TGM/src/main/java/network/warzone/tgm/modules/MatchResultModule.java#L63
In accordance with the provided footage, it is ignoring AMBIENT.

@jellz @jorgeberrex is this called anywhere else?
Thought I replaced these in the entire project.
